bundles/k3d/demo/uds-bundle.yaml

```diff
@@ -0,0 +1,157 @@
+kind: UDSBundle
+metadata:
+  name: k3d-swf-demo
+  description: A UDS bundle for deploying the UDS Software Factory with full uds-core on a development cluster
+  # x-release-please-start-version
+  version: 0.1.0
+  # x-release-please-end
+
+packages:
+  - name: uds-k3d-dev
+    repository: ghcr.io/defenseunicorns/packages/uds-k3d
+    # renovate: datasource=github-tags depName=defenseunicorns/uds-k3d versioning=semver
+    ref: 0.3.1
+
+  - name: init
+    repository: ghcr.io/defenseunicorns/packages/init
+    # renovate: datasource=github-tags depName=defenseunicorns/zarf versioning=semver
+    ref: v0.32.3
+
+  - name: uds-core #for now
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: "0.16.1-upstream"
```

Comment:
renovate?

Reply:
Correct me if im wrong but i should actually be able to delete these now with the updated renovate config you worked on right?

Reply:
yep